charts/images.yaml

```
@@ -208,15 +208,15 @@ images:
 - name: fluent-bit-plugin-installer
   sourceRepository: github.com/gardener/logging
   repository: eu.gcr.io/gardener-project/gardener/fluent-bit-to-loki
-  tag: "v0.35.0"
+  tag: "v0.36.0"
 - name: loki
   sourceRepository: github.com/grafana/loki
   repository: grafana/loki
   tag: "2.2.1"
 - name: loki-curator
   sourceRepository: github.com/gardener/logging
   repository: eu.gcr.io/gardener-project/gardener/loki-curator
-  tag: "v0.35.0"
+  tag: "v0.36.0"
 - name: kube-rbac-proxy
   sourceRepository: https://github.com/brancz/kube-rbac-proxy
```

If you are pushing any change, please remove the `https://` prefix.
```suggestion
  sourceRepository: github.com/brancz/kube-rbac-proxy
```